stps/sig-network/ip-request-stp.md

```diff
@@ -0,0 +1,216 @@
+| **Testability**                         | [V]   | Confirmed requirements are **testable and unambiguous**.                                                                                                                                 |           |
+| **Acceptance Criteria**                 | [V]   | Ensured acceptance criteria are **defined clearly** (clear user stories; D/S requirements clearly defined in Jira).                                                                      |           |
+| **Non-Functional Requirements (NFRs)**  | [ ]   | Confirmed coverage for NFRs, including Performance, Security, Usability, Downtime, Connectivity, Monitoring (alerts/metrics), Scalability, Portability (e.g., cloud support), and Docs.  |           |
+
+
+#### **2. Technology and Design Review**
+
+| Check                             | Done  | Details/Notes                                                                                                                                            | Comments                                                                                                                                            |
+|:--------------------------------- |:----- |:-------------------------------------------------------------------------------------------------------------------------------------------------------- |:--------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Developer Handoff/QE Kickoff**  | [V]   | A meeting where Dev/Arch walked QE through the design, architecture, and implementation details. **Critical for identifying untestable aspects early.**  |                                                                                                                                                     |
+| **Technology Challenges**         | [V]   | Identified potential testing challenges related to the underlying technology.                                                                            |                                                                                                                                                     |
+| **Test Environment Needs**        | [V]   | Determined necessary **test environment setups and tools**.                                                                                              |                                                                                                                                                     |
+| **API Extensions**                | [V]   | Reviewed new or modified APIs and their impact on testing.                                                                                               |                                                                                                                                                     |
```

Comment:
a new CRD is going to be added - should be included here and add if any testing is needed for it